xml/System.Diagnostics/Process.xml

```diff
@@ -5230,10 +5230,10 @@ The file specified in the <paramref name="fileName" /> could not be found.</exce
         <Parameter Name="arguments" Type="System.Collections.Generic.IEnumerable&lt;System.String&gt;" Index="1" FrameworkAlternate="net-5.0" />
       </Parameters>
       <Docs>
-        <param name="fileName">To be added.</param>
-        <param name="arguments">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="fileName">The name of a document or application file to run in the process.</param>
```

Comment:
@adamsitnik @eiriktsarpalis since you own Diagnostics.Process, can you review this file please?